tools/license-checker/src/main.rs

```diff
@@ -91,6 +91,8 @@ use ignore::WalkBuilder;
 use std::path::{Path, PathBuf};
 use std::process::exit;
 
+use colored::Colorize;
```

Comment:
I think you're trying to separate `use std` from non-`std` `use` statements, but they're all already clumped together anyway (see `use ignore::WalkBuilder`). Lets combine `use colored::...` with the above block of `use` statements.

Reply:
Moved use statements.